src/SprykerEco/Zed/Unzer/UnzerConfig.php

```diff
@@ -37,41 +38,124 @@ class UnzerConfig extends AbstractBundleConfig
+            UnzerSharedConfig::PAYMENT_METHOD_KEY_BANK_TRANSFER,
+            UnzerSharedConfig::PAYMENT_METHOD_KEY_MARKETPLACE_BANK_TRANSFER,
+        ],
+        UnzerConstants::UNZER_PAYMENT_METHOD_CREDIT_CARD => [
+            UnzerSharedConfig::PAYMENT_METHOD_KEY_CREDIT_CARD,
+            UnzerSharedConfig::PAYMENT_METHOD_KEY_MARKETPLACE_CREDIT_CARD,
+        ],
+        UnzerConstants::UNZER_PAYMENT_METHOD_DIRECT_DEBIT => [
+            UnzerSharedConfig::PAYMENT_METHOD_KEY_DIRECT_DEBIT,
+        ],
+        UnzerConstants::UNZER_PAYMENT_METHOD_DIRECT_DEBIT_SECURED => [
+            UnzerSharedConfig::PAYMENT_METHOD_KEY_DIRECT_DEBIT_SECURED,
+        ],
+        UnzerConstants::UNZER_PAYMENT_METHOD_ELECTRONIC_PAYMENT_STANDARD => [
+            UnzerSharedConfig::PAYMENT_METHOD_KEY_ELECTRONIC_PAYMENT_STANDARD,
+        ],
+        UnzerConstants::UNZER_PAYMENT_METHOD_GIROPAY => [
+            UnzerSharedConfig::PAYMENT_METHOD_KEY_GIROPAY,
+        ],
+        UnzerConstants::UNZER_PAYMENT_METHOD_IDEAL => [
+            UnzerSharedConfig::PAYMENT_METHOD_KEY_IDEAL,
+        ],
+        UnzerConstants::UNZER_PAYMENT_METHOD_INVOICE => [
+            UnzerSharedConfig::PAYMENT_METHOD_KEY_INVOICE,
+        ],
+        UnzerConstants::UNZER_PAYMENT_METHOD_INVOICE_SECURED => [
+            UnzerSharedConfig::PAYMENT_METHOD_KEY_INVOICE_SECURED,
+        ],
+        UnzerConstants::UNZER_PAYMENT_METHOD_PAYPAL => [
+            UnzerSharedConfig::PAYMENT_METHOD_KEY_PAYPAL,
+        ],
+        UnzerConstants::UNZER_PAYMENT_METHOD_POST_FINANCE_CARD => [
+            UnzerSharedConfig::PAYMENT_METHOD_KEY_POST_FINANCE_CARD,
+        ],
+        UnzerConstants::UNZER_PAYMENT_METHOD_POST_FINANCE_EFINANCE => [
+            UnzerSharedConfig::PAYMENT_METHOD_KEY_POST_FINANCE_EFINANCE,
+        ],
+        UnzerConstants::UNZER_PAYMENT_METHOD_PREPAYMENT => [
+            UnzerSharedConfig::PAYMENT_METHOD_KEY_PREPAYMENT,
+        ],
+        UnzerConstants::UNZER_PAYMENT_METHOD_PRZELEWY24 => [
+            UnzerSharedConfig::PAYMENT_METHOD_KEY_PRZELEWY24,
+        ],
+        UnzerConstants::UNZER_PAYMENT_METHOD_SOFORT => [
+            UnzerSharedConfig::PAYMENT_METHOD_KEY_SOFORT,
+            UnzerSharedConfig::PAYMENT_METHOD_KEY_MARKETPLACE_SOFORT,
+        ],
+        UnzerConstants::UNZER_PAYMENT_METHOD_WECHAT_PAY => [
+            UnzerSharedConfig::PAYMENT_METHOD_KEY_WECHAT_PAY,
+        ],
     ];
 
     /**
-     * @var array<string>
+     * @var array<array-key, string>
      */
     protected const AUTHORIZE_PAYMENT_METHODS = [
-        UnzerSharedConfig::PAYMENT_METHOD_MARKETPLACE_CREDIT_CARD,
+        UnzerSharedConfig::PAYMENT_METHOD_KEY_MARKETPLACE_CREDIT_CARD,
     ];
 
     /**
-     * @var array<string>
+     * @var array<string, string>
+     */
+    protected const UNZER_PAYMENT_METHOD_NAMES_MAP = [
+        UnzerSharedConfig::PAYMENT_METHOD_KEY_ALIPAY => UnzerSharedConfig::PAYMENT_METHOD_NAME_ALIPAY,
```

Comment:
Please confirm from Jeremy if we able to add all payment methods to config right now

Reply:
Confirmed by Jeremy:
> Yes you can add all of them, based on the API response, we will know which one the project will use